packages/renderer/src/lib/ui/DetailsPage.svelte

```diff
@@ -1,12 +1,34 @@
 <script lang="ts">
 import { DetailsPage } from '@podman-desktop/ui-svelte';
+import type { Snippet } from 'svelte';
 import { router } from 'tinro';
 
 import { currentPage, lastPage } from '../../stores/breadcrumb';
 
-export let title: string;
-export let titleDetail: string | undefined = undefined;
-export let subtitle: string | undefined = undefined;
+interface Props {
+  title: string;
+  titleDetail?: string;
+  subtitle?: string;
+
+  contentSnippet?: Snippet;
+  tabsSnippet?: Snippet;
+  iconSnippet?: Snippet;
+  subtitleSnippet?: Snippet;
+  actionsSnippet?: Snippet;
+  detailSnippet?: Snippet;
+}
+
+const {
+  title,
+  titleDetail = undefined,
+  subtitle = undefined,
+  contentSnippet: localContentSnippet,
```

Comment:
Wondering why we need different names here ?

Reply:
This is to avoid conflict in:
```
{#snippet iconSnippet()}{@render localIconSnippet?.()}{/snippet}
```

Or we would have the same name for the snippet of the current component and the snippet of the sub-component